tracpro/polls/models.py

```diff
@@ -472,7 +472,21 @@ def get_responses(self, region=None, include_subregions=True,
                 responses = responses.filter(contact__region=region)
         if not include_empty:
             responses = responses.exclude(status=Response.STATUS_EMPTY)
-        return responses.select_related('contact')
+
+        responses = responses.select_related('contact')
+
+        # Remove duplicate resopnse from the same user on the same day
+        response_tuple_list = []
```

Comment:
``response_tuple_list`` would be a tiny bit more efficient as a set, since we don't care about order or duplicates, and we're checking for membership in it a lot.